src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -148,7 +148,7 @@ private HttpRequestMessage CreateRequestMessage(Uri endpoint, IDictionary<string
 
                 if (cancellationToken.IsCancellationRequested)
                 {
-                    isRetryable = false;
+                    throw exception;
```

Comment:
Prefer to use `throw` instead. See https://stackoverflow.com/questions/730250/is-there-a-difference-between-throw-and-throw-ex

Also, consioder adding a `log.Info` for this.

Reply:
Done